### PR TITLE
Add `parse-args` function

### PR DESCRIPTION
**Pull request by squiter**
That function splits the `operator` from the `operands`.

### CODE REVIEW

**Comment by erickmazer** [comment]
👍

**Comment by wevtimoteo** [comment]
:clap:

**Comment by wevtimoteo** [comment]
Now :rocket: :tada:

File: parsers.clj [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+(defn format-float
+  [float-number]
+  (format "%.2f" float-number))
+
+(defn call
+  [[operator args]]
+
+  (format-float
+    (reduce
```
Comment (squiter) [inline parsers.clj:18]: :+1: :heart:

**Comment by squiter** [comment]
:+1: :ship:

**Comment by wevtimoteo** [comment]
First working `clj` program

**Pull request merged**